muxers/mplex/Cargo.toml

```diff
@@ -3,7 +3,7 @@ name = "libp2p-mplex"
 edition = "2021"
 rust-version = "1.56.1"
 description = "Mplex multiplexing protocol for libp2p"
-version = "0.33.0"
+version = "0.34.0"
```

Comment:
```suggestion
version = "0.33.1"
```

Is this a breaking change?

Reply:
Yes because `Substream` is a public type and has type parameter now!

Reply:
Oh, good point. Thanks!